src/components/CodeEditorHeader.tsx

```diff
@@ -76,6 +83,68 @@ export const CodeEditorHeader = ({
       })
     }
   }
+
+  const importRegex =
+    /import\s+(?:\*\s+as\s+)?([a-zA-Z_$][\w$]*)\s+from\s+["']\.\/manual-edits\.json["'];?/
```

Comment:
this code is too specific, it doesn't belong here. Create a utility function called `checkIfManualEditsImported` in a separate file